pkg/web/src/app/Providers/components/ProvidersTable/ProviderActionsDropdown.tsx

```diff
@@ -11,6 +16,21 @@ import { isSameResource } from '@app/queries/helpers';
 import { useHistory } from 'react-router-dom';
 import { useClusterProvidersQuery } from '@app/queries';
 
+export const hasRunningMigration = ({
+  plans = [],
+  providerMetadata,
+}: {
+  plans: IPlan[];
+  providerMetadata: INameNamespaceRef;
+}) =>
+  !!plans
+    .filter((plan) => hasCondition(plan.status?.conditions || [], 'Executing'))
+    .find((runningPlan) => {
+      const { source, destination } = runningPlan.spec.provider;
+      return (
+        isSameResource(providerMetadata, source) || isSameResource(providerMetadata, destination)
+      );
+    });
```

Comment:
nit-pick: blank line between definitions?

Reply:
Right. This is due to VS Code plugin that displays git info (commit messages, etc) above/next to code.